src/main/java/org/geysermc/discordbot/commands/GithubCommand.java

```diff
@@ -0,0 +1,68 @@
+package org.geysermc.discordbot.commands;
+
+import com.jagrosh.jdautilities.command.SlashCommand;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import org.geysermc.discordbot.util.BotColors;
+import org.geysermc.discordbot.util.BotHelpers;
+import org.geysermc.discordbot.util.MessageHelper;
+import org.kohsuke.github.*;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Objects;
+
+public class GithubCommand extends SlashCommand {
+
+    public GithubCommand() {
+        this.name = "github";
+        this.arguments = "<repo>";
+        this.help = "Get info about a given GitHub repo.";
+        this.guildOnly = false;
+        this.options = Collections.singletonList(
+                new OptionData(OptionType.STRING, "repo", "The repository to lookup, defaults to GeyserMC/Geyser")
+                        .setRequired(true)
+        );
+    }
+
+    @Override
+    protected void execute(SlashCommandEvent event) {
+        String repo = Objects.requireNonNull(event.getOption("repo")).getAsString();
+        event.deferReply(false).queue(interactionHook -> {
+            try {
+                interactionHook.editOriginalEmbeds(handle(repo)).queue();
+            } catch (IOException ignored) { }
```

Comment:
We shouldn't ignore this or, in the off chance one occurs, the bot will never reply. This could be a generic error embed with "Could not retrieve data from GitHub, try again later!"